modules/rhdh-install-web-console.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * layered-products-web-console.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rhdh-install-web-console_{context}"]
+=  Installing the {rh-dev-hub} using the {product-title} web console
+
+The web console provides instructions in a quick start to install the {rh-dev-hub} Operator.
+
+.Prerequisites
+* You must be logged into the {product-title} web console with `admin` privileges.
+
+.Procedure
+. On the *Overview* page of the Administrator perspective, click *Install {rh-dev-hub} (RHDH) with an Operator* in the *Getting started resources* tile.
+. A quick start pane is displayed with instructions for you to install the {rh-dev-hub} with an Operator. Follow the quick start to install the Operator, create a {rh-dev-hub} Instance, and add your Instance to the OpenShift Console Application menu.
```

Comment:
Is that menu referenced part of the UI. I wasn't sure if it should be **OpenShift Console Application** menu.